code/modules/detectivework/detective_work.dm

```diff
@@ -100,3 +100,18 @@
 	A.add_fingerprint_list(return_fingerprints())
 	A.add_hiddenprint_list(return_hiddenprints())
 	A.fingerprintslast = fingerprintslast
+
+/obj/item/card/assistant_disguise_card
+	name = "assistant disguise card"
+	desc = "A small card, that when used on any ID, will change the job role to Assistant whilst retaining current access."
+	icon_state = "data_1"
```

Comment:
This is really bad idea because it literally removes player-player interaction. Why would you ever ask HoP for anything?

Reply:
If this gives the Detective the same name ID, but different role... why give the Detective a voice changer? So they can talk like an assistant?